### PR TITLE
Create finance, SaaS, and internet/entertainment page under use-cases

### PR DESCRIPTION
**Pull request by sandylcruz**
* Fixes https://github.com/gravitational/next/issues/517, https://github.com/gravitational/next/issues/516, https://github.com/gravitational/next/issues/518
* Add Finance, SaaS, and Internet-Entertainment Pages

### CODE REVIEW

**Comment by C-STYR** [comment]
Looking good so far!

File: pages/solutions/finance/index.mdx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    subtitle={meta.subtitle}
+    description={meta.description}
+    width="100%"
+  >
+    <NextImage
+      src="./assets/powering-the-world@2x.png"
+      alt="teleport powering the world"
+      height={418.37}
+      width={600}
+    />
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply pages/solutions/finance/index.mdx:56]: Agreed. I have updated these.

File: pages/solutions/internet-entertainment/index.mdx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  - twitch
+  - rushstreet
+  - epicgames
+  - wish
+  - coupang
+  - thrasio
+  - tokopedia
+  - bigcommerce
+  - thredup
+  - theshadestore
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply pages/solutions/internet-entertainment/index.mdx:21]: removed

File: pages/solutions/finance/index.mdx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  - gitlab
+  - gusto
+  - hpe
+  - ibm
+  - nasdaq
+  - samsung
+  - sumologic
+  - swisscom
+  - threatstack
+  - twitch
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply pages/solutions/finance/index.mdx:26]: removed

File: pages/solutions/finance/index.mdx [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+$ tsh login --proxy=example.com
+
+# on a server
+
+$ apt install teleport
+
+# in a Kubernetes cluster
+
+$ helm install
+
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply pages/solutions/finance/index.mdx:204]: fixed. just FYI, this happens automatically if you save your work in an mdx file, also the three backticks at the bottom that you mention later on.

File: pages/solutions/finance/index.mdx [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+  - nasdaq
+  - samsung
+  - sumologic
+  - swisscom
+  - threatstack
+  - twitch
+---
+
+import {
+  AnimationHeader,
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply pages/solutions/finance/index.mdx:30]: fixed

File: pages/use-cases/e-commerce/index.mdx [hunk 6]
```diff
@@ -0,0 +1,234 @@
+---
+title: A modern access solution for ecommerce & entertainment
+subtitle: Teleport Access Plane
+description: Teleport’s access solution empowers teams to grow & innovate, without worrying about security or getting slowed down by complex security systems.
+reviews:
+  - glu
+---
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply pages/use-cases/e-commerce/index.mdx]: agreed

File: components/CenterCard/CenterCard.tsx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+            borderRadius="circle"
+            height={48}
+            width={48}
+            boxShadow="0px 8px 16px rgba(12, 12, 14, 0.24)"
+            mb={[2, 4]}
+          >
+            <NextImage src={src} alt="small icon" height={24} width={24} />
+          </Flex>
+        )}
+        <Flex
```
Comment (Alqanar) [inline components/CenterCard/CenterCard.tsx:40]: I think we need use props `as` here.
Inserting text into `div` we should use only in extraordinary situation
`as = h2`

File: components/CenterCard/CenterCard.tsx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+          fontSize={["text-md", "header-1"]}
+          lineHeight="xxl"
+          mb={[2, 5]}
+          fontWeight="black"
+          flexWrap="wrap"
+        >
+          {title}
+        </Flex>
+        <Flex
+          color="black"
```
Comment (Alqanar) [inline components/CenterCard/CenterCard.tsx:53]: I think we need use props `as` here.
Inserting text into `div` we should use only in extraordinary situation
`as = p`

**Review by Alqanar — APPROVED** [review]
There are still changes here that I would like to see

File: components/CenterCard/CenterCard.tsx (hunk 7 above)
Comment (C-STYR) [reply components/CenterCard/CenterCard.tsx:40]: changed.

File: components/CenterCard/CenterCard.tsx (hunk 8 above)
Comment (C-STYR) [reply components/CenterCard/CenterCard.tsx:53]: changed.

**Comment by ardeche07** [comment]
from Michael: I updated the H1 and added a description to the finance page. The other changes I'd like to see on finance page are:

For the logos at the top Nasdaq, square, moody’s, crypto.com, wise, may street

Slow the transition on the quote slider.  Moves a little fast for slow readers like 

The greyed out logos under the images are too knocked back.  We can keep greyscale here, but can you increase the contrast?

Add Crypto.com logo to blockchain section.

Remove our vision of the future section. We don’t want to use that any more. Will be replaced eventually by something more appropriate.

File: pages/use-cases/saas/index.mdx [hunk 9]
```diff
@@ -0,0 +1,219 @@
+---
+title: Grow your SaaS business by easily demonstrating security and compliance to your customers.
+subtitle: Software-as-a-service (SaaS) Providers
+hideWave: true
+meta-description: Teleport’s Access Plane empowers leading financial institutions to innovate, without sacrificing speed for security.
+reviews:
+  - elastic
```
Comment (ardeche07) [inline pages/use-cases/saas/index.mdx:7]: should Carta be added here? I don't know what the reviews field does but Carta is on the page as well

**Comment by ardeche07** [comment]
Ok, I finished by review. I added copy to all the pages.

For some reason the description/h2 is not showing up on the SaaS page.  Not sure why.  I think it looks like the other pages where it is working.

With the exception of the things I pointed out in the comment above, the only other change needed is to increase the contrast on all the knocked back logos in the 3 main sections on each page. Some of these are really hard to read.  These are looking great!

File: pages/use-cases/saas/index.mdx (hunk 9 above)
Comment (C-STYR) [reply pages/use-cases/saas/index.mdx:8]: @ardeche07 the review field specifies which quotes appear in the sliding quote card component, which you will see on the index.mdx page as <Feedback/>.

**Comment by C-STYR** [comment]
> For some reason the description/h2 is not showing up on the SaaS page. Not sure why. I think it looks like the other pages where it is working.


In the component itself (here called <SectionHeader/>), you will need to pass in a `description` prop. Please note this is different than the description field in the top section.  `description` in the top section is the meta-description (I've updated these to reflect what I think are your intentions).  You can set the text that appears as the header description by changing the value of the `description` prop in `SectionHeader`.  To distinguish it from the meta-description I've given it the name `headerDescription`.  Thus now in `SectionHeader`, `description={meta.headerDescription`}.

**Comment by C-STYR** [comment]
@annabambi please see @ardeche07 's comments about the logo stripes and increasing contrast/lessening opacity and also adding in crypto.com to blockchain section strip in the finance section...thanks!

**Comment by ardeche07** [comment]
Can we please add Goldman Sachs logo to the finserv page @annabambi

**Pull request merged**